### ISSUE TITLE
during transcribe got error

### ISSUE THREAD

**Issue by yilihtien** [alibaba-damo-academy/FunASR#1372]
## 🐛 Bug

當檔案是.wav pcm_s16le 時長02:59:13，file size:984MB.錯誤如下：

1File "/home/y6p2/mambaforge/envs/envChat/lib/python3.10/site-packages/funasr/auto/auto_model.py", line 206, in generate
    return self.inference_with_vad(input, input_len=input_len, **cfg)
  File "/home/y6p2/mambaforge/envs/envChat/lib/python3.10/site-packages/funasr/auto/auto_model.py", line 394, in inference_with_vad
    labels = self.cb_model(spk_embedding.cpu(), oracle_num=self.preset_spk_num)
  File "/home/y6p2/mambaforge/envs/envChat/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/y6p2/mambaforge/envs/envChat/lib/python3.10/site-packages/funasr/models/campplus/cluster_backend.py", line 162, in forward
    labels = self.umap_hdbscan_cluster(X)
  File "/home/y6p2/mambaforge/envs/envChat/lib/python3.10/site-packages/funasr/models/campplus/cluster_backend.py", line 127, in __call__
    ).fit_transform(X)
  File "/home/y6p2/mambaforge/envs/envChat/lib/python3.10/site-packages/umap/umap_.py", line 2887, in fit_transform
    self.fit(X, y, force_all_finite)
  File "/home/y6p2/mambaforge/envs/envChat/lib/python3.10/site-packages/umap/umap_.py", line 2608, in fit
    ) = nearest_neighbors(
  File "/home/y6p2/mambaforge/envs/envChat/lib/python3.10/site-packages/umap/umap_.py", line 343, in nearest_neighbors
    knn_indices, knn_dists = knn_search_index.neighbor_graph
  File "/home/y6p2/mambaforge/envs/envChat/lib/python3.10/site-packages/pynndescent/pynndescent_.py", line 1615, in neighbor_graph
    self._distance_correction(self._neighbor_graph[1]),
numpy.core._exceptions._UFuncNoLoopError: ufunc 'correct_alternative_cosine' did not contain a loop with signature matching types <class 'numpy.dtype[float32]'> -> None

當檔案是mp3 時長02:59:13，file size:164MB.錯誤如下：
File "/home/y6p2/y6p2chat_dev/start.py", line 504, in filetranscribe
    res = s2tmodel.generate(input=file,
  File "/home/y6p2/mambaforge/envs/envChat/lib/python3.10/site-packages/funasr/auto/auto_model.py", line 206, in generate
    return self.inference_with_vad(input, input_len=input_len, **cfg)
  File "/home/y6p2/mambaforge/envs/envChat/lib/python3.10/site-packages/funasr/auto/auto_model.py", line 394, in inference_with_vad
    labels = self.cb_model(spk_embedding.cpu(), oracle_num=self.preset_spk_num)
  File "/home/y6p2/mambaforge/envs/envChat/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/y6p2/mambaforge/envs/envChat/lib/python3.10/site-packages/funasr/models/campplus/cluster_backend.py", line 162, in forward
    labels = self.umap_hdbscan_cluster(X)
  File "/home/y6p2/mambaforge/envs/envChat/lib/python3.10/site-packages/funasr/models/campplus/cluster_backend.py", line 127, in __call__
    ).fit_transform(X)
  File "/home/y6p2/mambaforge/envs/envChat/lib/python3.10/site-packages/umap/umap_.py", line 2887, in fit_transform
    self.fit(X, y, force_all_finite)
  File "/home/y6p2/mambaforge/envs/envChat/lib/python3.10/site-packages/umap/umap_.py", line 2608, in fit
    ) = nearest_neighbors(
  File "/home/y6p2/mambaforge/envs/envChat/lib/python3.10/site-packages/umap/umap_.py", line 343, in nearest_neighbors
    knn_indices, knn_dists = knn_search_index.neighbor_graph
  File "/home/y6p2/mambaforge/envs/envChat/lib/python3.10/site-packages/pynndescent/pynndescent_.py", line 1615, in neighbor_graph
    self._distance_correction(self._neighbor_graph[1]),
numpy.core._exceptions._UFuncNoLoopError: ufunc 'correct_alternative_cosine' did not contain a loop with signature matching types <class 'numpy.dtype[float32]'> -> None

程式如下：
1. s2tmodel = AutoModel(model="paraformer-zh", model_revision="v2.0.4",
                  vad_model="fsmn-vad", vad_model_revision="v2.0.4",
                  punc_model="ct-punc-c", punc_model_revision="v2.0.4",
                  spk_model="cam++", spk_model_revision="v2.0.2", 
                  )
2. res = s2tmodel.generate(input=file, 
                     batch_size_s=300, 
                     hotword='魔搭')

如果將wav檔切成三份，還是有問題。將mp3檔切成3份就能順利轉譯。
### Environment

 - OS (e.g., Linux):ubuntu 20.04
 - FunASR Version (e.g., 1.0.5):
 - ModelScope Version (e.g., 1.12.0):
 - PyTorch Version (e.g., 2.0.1):
 - How you installed funasr (`pip`, source):pip
 - Python version:3.10.13
 - GPU (e.g., V100M32)A100 VM passthrough 80G
 - CUDA/cuDNN version (e.g., cuda12.2):cuda12.2
 - Docker version (e.g., funasr-runtime-sdk-cpu-0.4.1)
 - Any other relevant information:

**Comment by yilihtien** [comment]
install funasr new version
add preset_spk_num
